content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -0,0 +1,156 @@
+
+#### Example using XFF capability with httpbin
+
+1. Specify `numTrustedProxies` as 2 either through `MeshConfig` or an `proxy.istop/io/config` annotation.
+2. Create `httpbin` namespace
+    {{< text bash >}}
+    $ kubectl create namespace httpbin
+    namespace/httpbin created
+    {{< /text >}}
+3. Set the `istio-injection` label to `enabled` for sidecar injection
+    {{< text bash >}}
+    $ kubectl label --overwrite namespace httpbin istio-injection=enabled
+    namespace/httpbin labeled
+    {{< /text >}}
+4. Deploy `httpbin` in the `httpbin` namespace
+    {{< text bash >}}
+    $ kubectl apply -n httpbin -f samples/httpbin/httpbin.yaml
+    {{< /text >}}
+5. Deploy a gateway associated with `httpbin`
+    {{< text bash >}}
+    $ kubectl apply -n httpbin -f samples/httpbin/httpbin-gateway.yaml
+    {{< /text >}}
+6. Set a local `$GATEWAY_URL` environmental variable based on your Istio ingress gateway's IP address
+    {{< text bash >}}
+    $ export GATEWAY_URL=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+    {{< /text >}}
+7. Run the following `curl` command to verify the `X-Envoy-External-Address` and `X-Forwarded-For` are set correctly.
+    {{< text bash >}}
+    $ curl -H 'X-Forwarded-For: 56.5.6.7, 72.9.5.6, 98.1.2.3' $GATEWAY_URL/get?show_env=true
+    {
+      "args": {
+        "show_env": "true"
+      },
+      "headers": {
+        ...
+        "X-Envoy-External-Address": "72.9.5.6",
+        ...
+        "X-Forwarded-For": "56.5.6.7, 72.9.5.6, 98.1.2.3, <YOUR GATEWAY IP>",
+        ...
+      },
+      ...
+    }    
+    {{< /text >}}
+
+### Configuring X-Forwarded-Client-Cert Headers
+
+From [Envoy's documenation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-client-cert)
+regarding XFCC:
+> x-forwarded-client-cert (XFCC) is a proxy header which indicates certificate information of part or all of the clients
+> or proxies that a request has flowed through, on its way from the client to the server. A proxy may choose to
+> sanitize/append/forward the XFCC header before proxying the request.
+
+To configure how XFCC Headers are handled add the following to your `topology.yaml` file.
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      gatewayTopology:
+        forwardClientCertDetails: <ENUM_VALUE>
+{{< /text >}}
+
+where `ENUM_VALUE` can be of the following type.
+
+| ENUM_VALUE          |                                                                                                                                |
+|---------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| SANITIZE            | Do not send the XFCC header to the next hop. This is the default value.                                                        |
```

Comment:
In Istio we also have `UNDEFINED`

Additionally, let's specify that the default value for ingress gateway is `SANITIZE_SET`

Reply:
added and clarified